sormas-app/app/src/main/java/de/symeda/sormas/app/person/edit/PersonEditFragment.java

```diff
@@ -164,6 +166,7 @@ public static void setUpLayoutBinding(
 			false);
 
```

Comment:
We have the same problem as in [https://github.com/hzi-braunschweig/SORMAS-Project/pull/3636#discussion_r533942951](https://github.com/hzi-braunschweig/SORMAS-Project/pull/3636#discussion_r533942951). The salutation also shows in the app for non german servers.
We should either find a fix, or use the workaround as in the linked pull request.